ddl/rollingback.go

```diff
@@ -60,8 +60,8 @@ func convertAddIdxJob2RollbackJob(d *ddlCtx, t *meta.Meta, job *model.Job, tblIn
 		}
 	}
 
-	// the second args will be used in onDropIndex.
-	job.Args = []interface{}{indexInfo.Name, getPartitionIDs(tblInfo)}
+	// the second and the third args will be used in onDropIndex.
+	job.Args = []interface{}{indexInfo.Name, false /* ifExists */, getPartitionIDs(tblInfo)}
```

Comment:
Won't this break the compatibility, if it is inserted into the middle, @tangenta ?

Reply:
For `DropColumn()`, the `exists` argument is the last one... there are two cases:

- New version TiDB creates a job (args: [idx_name, ifExists]), old version TiDB runs the job(ifExists is ignored). 
- Old version TiDB creates a job(args: [idx_name]), new version TiDB runs the job(ifExists is decoded to `false`).

In the worse case, the DDL owner TiDB reports an error. The data is consistent.